spec/lib/twingly/url_spec.rb

```diff
@@ -102,6 +114,16 @@ def valid_urls
       end
     end
 
+    context "IDNA 2003 vs IDNA 2008" do
```

Comment:
I know the wording doesn't follow what we have... it is hard writing these things. In the code, you have "it", but in the spec output, you don't have "it"... so should you take it into consideration or not?

Now I just used `context` as a "title" and then added an sentence that would read well (IMHO) in the spec output:

```
    IDNA 2003 vs IDNA 2008
      if you normalize a URL with the domain "straße.de"
        the domain of the normalized URL should be "xn--strae-oqa.de"
```